src/utils/uploads.js

```diff
@@ -335,11 +337,17 @@ function getDataTransferItemId(
 }
 
 /**
- * Multiput uploads require the use of crypto, which is only supported in secure contexts
+ * Multiput uploads require the use of crypto, which is only supported in secure contexts.
+ * Multiput uploads is not supported on mobile iOS Safari.
  */
 function isMultiputSupported(): boolean {
     const cryptoObj = window.crypto || window.msCrypto;
-    return window.location.protocol === 'https:' && cryptoObj && cryptoObj.subtle;
+
+    if (Browser.isMobile() && Browser.isSafari()) {
```

Comment:
Even though Mobile Safari versions 11.0 & 11.1 work with multiput uploads, we are turning it off for all since user agent sniffing of the version number could be unreliable.